docs/guide/docs/channels/telegram.md

```diff
@@ -5,12 +5,25 @@ title: Telegram
 
 ## Requirements
 
-- Use Telegram's BotFather to create a bot. Please refer to [How do I create a bot?](https://core.telegram.org/bots#3-how-do-i-create-a-bot)
-- Have your Telegram bot token ready
+The process for connecting to Telegram is simple. Firstly, you need to create a bot on Telegram. Thereafter, you can generate and access an authorization token, which you will use to connect your Telegram bot to your Botpress chatbot.  
+
+### Create a bot
+To create a bot on Telegram, use Telegram's BotFather.The BotFather will ask you for a name and username, then generate an authorization token for your new bot.
+
+The name of your bot is displayed in contact details and elsewhere.
+
+The Username is a short name to be used in mentions and t.me links. Usernames are 5-32 characters long and are case insensitive but may only include Latin characters, numbers, and underscores. Your bot's username must end in 'bot', e.g. 'tetris_bot' or 'TetrisBot'.
 
 ## Setup
 
-1. Create or edit this file `data/bots/<your_bot>/config/channel-telegram.json` and enter your Telegram bot token:
+### Generate an Authorisation Token
+When you create a Telegram bot, Botfather will automatically generate a token. The token is a string along the lines of 110201543:AAHdqTcvCH1vGWJxfSeofSAs0K5PALDsaw that is required to authorize the bot and send requests to the Bot API. Keep your token secure and store it safely; anyone can use it to control your bot.
```

Comment:
is this one a placeholder as well ? if not I suggest you go ahead and revoke this token quickly

Reply:
Done